src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Greeting from './pages/Greetings';
+
+function App() {
+  return (
+    <Greeting />
+  );
+}
```

Comment:
- [ ] the App component should have a route that leads to the Greeting component.